source/Cosmos.System2_Plugs/System/Threading/ThreadImpl.cs

```diff
@@ -1,13 +1,11 @@
 using System;
-using System.Threading;
-
 using Cosmos.HAL;
-
 using IL2CPU.API.Attribs;
 
 namespace Cosmos.System_Plugs.System.Threading
 {
-    [Plug(Target = typeof(Thread))]
+    //[Plug(Target = typeof(Thread))]
+    [Plug(TargetName = "System.Threading.Thread")]
```

Comment:
Why doesn't `Target = typeof(Thread)` work? If there are different `Thread` classes in different assemblies, you should specify the assembly to make it clear which type is being plugged, like `System.Threading.Thread, System.Private.CoreLib`.